dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -558,7 +558,30 @@
       <BatchMergeXaml RS1Pages="@(RS1StylePage)" RS2Pages="@(RS2StylePage)"  RS3Pages="@(RS3StylePage)" 
                       RS4Pages="@(RS4StylePage)" RS5Pages="@(RS5StylePage)"  N19H1Pages="@(NineteenH1StylePage)" 
                       PostfixForGeneratedFile="generic" OutputDirectory="$(OutDir)" />
-       <Copy SourceFiles="$(OutDir)rs1_generic.xaml" DestinationFiles="$(OutDir)Generic.xaml" />
+      <Copy SourceFiles="$(OutDir)rs1_generic.xaml" DestinationFiles="$(OutDir)Generic.xaml" />
+
+      <!-- NB: We have to use CreateProperty here instead of PropertyGroup.	
+      PropertyGroup values are always evaluated even when their enclosing target is skipped,	
+      whereas CreateProperty has the TaskParameter ValueSetByTask that can be used	
+      to only set a property if the target actually runs. -->	
+      <CreateProperty Value="True">	
+        <Output TaskParameter="ValueSetByTask" PropertyName="GenericXamlFileNeedsCompilation" />	
+      </CreateProperty>	
+      <ItemGroup>	
+        <FileReads Include="@(RS1StylePage)" />
+        <FileReads Include="@(RS2StylePage)" />
```

Comment:
Are these FileReads and FileWrites necessary? What are they used for? The Target already has Inputs and Outputs attributes specified

Reply:
I made a clean build and then verify that. yes. we don't need that. 
I didn't add that before, then vs didn't do the job as I expect. then I copied FileReads/Writes from old file, and magic happened: it works. I thought maybe some body workaround the problem in this way.